2025/05.md

```diff
@@ -119,6 +119,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     | 2.7 | 30m | [Upsert](https://github.com/tc39/proposal-upsert) for stage 3 ([slides](https://docs.google.com/presentation/d/15J_tgYqrh-aPat0klS78BcDVGYJ88HOCC_SBsYkR4QY/))  | Daniel Minor |
     | 2.7 | 30m | [Iterator Sequencing](https://github.com/tc39/proposal-iterator-sequencing) for Stage 3 ([slides](https://docs.google.com/presentation/d/1XXAqt72dHIBQBvTRmR4UqOWbEcj_zqoS61NRMwriJ7s)) | Michael Ficarra |
     | 2 | 30m | [Iterator Chunking](https://github.com/tc39/proposal-iterator-chunking) for Stage 2.7 ([slides](https://docs.google.com/presentation/d/1Mse7PDM0vcMg4Ag_SK1_OGwVwrJuVxKz-qaWj2RyX8o)) | Michael Ficarra |
+    | 2 | 30m | [SeededPRNG](https://github.com/tc39/proposal-seeded-random) for stage 2 ([slides](https://docs.google.com/presentation/d/1wCKZx60SxFkhLHPrOHhzTWDrtr-OiE8p7oZWjGtSdmY/edit?usp=sharing)) | Tab Atkins-Bittner |
```

Comment:
This is listed as Stage 2 but going for Stage 2? I think this proposal is Stage 1.

Reply:
Fixed in ab1365ea

Reply:
Thanks, I'd misunderstood the purpose of the column.